default.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs ? import <nixpkgs> { } }:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "tabry-rs";
```

Comment:
The cargo package (and thus, resulting binary) is just called tabry, as it is intended to replace the old tabry executable (tabry ruby is just a library). Not sure what this should be called. I'm also not sure how I'm going to name the repos long term.

Reply:
Ok, I'll change it to `tabry`, I don't think it matters too much unless we wanted to move it to nixpkgs. When I import it into my home-manager config, I import it as "`tabry`"

Reply:
Have you ever gotten anything (either your own program or someone else's) in nixpkgs? I wonder what the process is like.
If I get a few more people using tabry, I may try :) Although higher priority would probably be brew since that is probably more common...